src/Ferry.js

```diff
@@ -1,62 +1,64 @@
 'use strict';
 
-import path from 'path';
-import Storage from './Storage';
 import Router from './Router';
```

Comment:
My logic was Ferry -> everything else, but I have no real preference. I can happily adjust to yours; glad to see we both like it organized! :)